engine/tests/phpunit/Elgg/Mocks/Database/MetastringsTable.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Elgg\Mocks\Database;
+
+class MetastringsTable extends \Elgg\Database\MetastringsTable {
+
+	/**
+	 * @var int
+	 */
+	public $iterator = 0;
+	
+	/**
+	 * {@inheritdoc}
+	 */
+	public function __construct(\Elgg\Cache\Pool $cache, \Elgg\Database $db) {
```

Comment:
Ah, you mean the constructor. I am keeping them in for now, until we update all `private` properties to `protected`.